backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/isShowQRBill.sql

```diff
@@ -0,0 +1,17 @@
+DROP FUNCTION IF EXISTS report.isShowQRBill(IN p_AD_Client_ID numeric, IN p_AD_Org_ID numeric);
+
+create or replace function report.isShowQRBill(IN p_AD_Client_ID numeric, IN p_AD_Org_ID numeric) RETURNS CHAR AS
+$$
+DECLARE
+    isShowQRBill CHAR DEFAULT 'Y';
+BEGIN
+ 
+        SELECT VALUE
+        INTO isShowQRBill
```

Comment:
clarified with cris, if there is no value, it will stay `'Y'` and not be reassigned `NULL`